rclpy/test/test_time_source.py

```diff
@@ -67,6 +68,19 @@ def publish_reversed_clock_messages(self):
             rclpy.spin_once(self.node, timeout_sec=1, executor=executor)
             time.sleep(1)
 
+    def set_use_sim_time_parameter(self, value):
+        self.node.set_parameters(
+            [Parameter('use_sim_time', Parameter.Type.BOOL, value)])
+        executor = rclpy.executors.SingleThreadedExecutor(context=self.context)
+
+        while rclpy.ok(context=self.context):
+            use_sim_time_param = self.node.get_parameter('use_sim_time')
+
+            if use_sim_time_param._type_ == Parameter.Type.BOOL:
+                break
```

Comment:
Suggestion: return a bool indicating if this method call was successful, then we can assert there wasn't an error in the tests. E.g. instead of breaking,

```python
return use_sim_time_param.value == value
```

Then when calling in the tests:

```python
assert set_use_sim_time_parameter(False)
```

Reply:
Fixed